### PR TITLE
DC-456: Add export endpoint to openapi.yml

### PR DESCRIPTION
**Pull request by fboulnois**

### CODE REVIEW

File: common/openapi.yml
Comment (fboulnois) [inline common/openapi.yml:149]: Is a post the right call here?

File: common/openapi.yml
Comment (fboulnois) [inline common/openapi.yml:148]: Thoughts about this endpoint?

File: common/openapi.yml
Comment (fboulnois) [inline common/openapi.yml:162]: Do we need to capture a response on success?

File: common/openapi.yml
Comment (fboulnois) [inline common/openapi.yml:165]: Do we need to capture a more specific error?

File: common/openapi.yml
Comment (fboulnois) [inline common/openapi.yml:387]: I was thinking of strongly typing the WorkspaceId, so it couldn't accidentally be defined as another type of uuid.

File: common/openapi.yml
Comment (pshapiro4broad) [reply common/openapi.yml:148]: I think `export` makes sense. (vs `link` which we aren't planning to do at this point).

File: common/openapi.yml
Comment (pshapiro4broad) [reply common/openapi.yml:149]: I think so, I think it makes more sense than `PUT` for this operation.

File: common/openapi.yml
Comment (pshapiro4broad) [reply common/openapi.yml:162]: Probably not.. if we were creating a workspace then returning the ID would make sense, but not if we're accepting the ID.

File: common/openapi.yml
Comment (pshapiro4broad) [inline common/openapi.yml:155]: For a required argument, is there a reason to use a body instead of a path argument? It seems equivalent to me, unless the body is large and requires a multipart attachment. Using a path argument makes it a little easier to use from scripts and swagger, IMO.

File: common/openapi.yml
Comment (s-rubenstein) [reply common/openapi.yml:160]: I think we probably want to throw a 403 if a user has 'Discoverer' but not 'Reader' access?

**Review by s-rubenstein — APPROVED** [review]
LGTM (Pending change)

**Review by tcjiang9 — APPROVED** [review]

File: common/openapi.yml
Comment (pshapiro4broad) [reply common/openapi.yml:162]: Maybe, but TDC shouldn't do any permission check here, it should leave it up to the storage system to do.

Comment (s-rubenstein) [reply common/openapi.yml:162]: Agreed that we shouldn't call to any permission check endpoint, but re: what the users of the API should expect, we want to provide possible error cases in our documentation.

**Pull request merged**
